mne/preprocessing/stim.py

```diff
@@ -4,10 +4,36 @@
 
 import numpy as np
 from scipy import signal, interpolate
+from ..evoked import Evoked
+from ..epochs import Epochs
+from ..io import Raw
+from ..utils import deprecated
 
 from .. import pick_types
+from ..io.pick import pick_channels
 
 
+def _get_window(start, end):
```

Comment:
can you add a minimal docstring describing what it does? It'll be useful for the developers